### DISCUSSION TITLE
need help with profile issues

### DISCUSSION THREAD

**Discussion by deerfoxl0l** [community/community#41132] [New to GitHub] [answered]
i have no idea how to make the clock on my profile go off army time![image](https://user-images.githubusercontent.com/115039465/206551778-bc4bc8a8-32cc-4d52-a65a-058864c9477c.png)

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! That is the default and sole option available at this moment - you can disable the time in your settings or by clicking the edit profile button and unchecking the box to display local time.

**Reply by deerfoxl0l** [reply]
thanks for replying

**Comment by dersav-joscon-gIqky1** [comment]
> i have no idea how to make the clock on my profile go off army time![image](https://user-images.githubusercontent.com/115039465/206551778-bc4bc8a8-32cc-4d52-a65a-058864c9477c.png)

**Comment by OranItzik** [comment]
hello,
please try to charge the account again ASAP
we update the payment method

Thanks
Shiri.

**Comment by 124688Droy** [comment]
I opened a ticket to have my email removed from my account so I can open a new account.  This has still not be corrected as I am not able to add my email to my new account.
this is the email I received from github 
Hey 124688DR!

An email address ([Danielle.Roy@aa.com](mailto:Danielle.Roy@aa.com)) was removed from your account. Visit [https://urldefense.com/v3/__https://github.com/settings/emails__;!!OgnrLJOUc50AAzM!MhG52oeXB945a1KVue1Y16i78MJrqVlhH_8dv5hJiRTLOakL4K4_p6KkkBpSFQhjE0Jz-mlX9yhg8OE$](https://urldefense.com/v3/__https:/github.com/settings/emails__;!!OgnrLJOUc50AAzM!MhG52oeXB945a1KVue1Y16i78MJrqVlhH_8dv5hJiRTLOakL4K4_p6KkkBpSFQhjE0Jz-mlX9yhg8OE$)  to review email addresses currently associated with your account.

To see this and other security events for your account, visit [https://urldefense.com/v3/__https://github.com/settings/security-log__;!!OgnrLJOUc50AAzM!MhG52oeXB945a1KVue1Y16i78MJrqVlhH_8dv5hJiRTLOakL4K4_p6KkkBpSFQhjE0Jz-mlXpmnkASQ$](https://urldefense.com/v3/__https:/github.com/settings/security-log__;!!OgnrLJOUc50AAzM!MhG52oeXB945a1KVue1Y16i78MJrqVlhH_8dv5hJiRTLOakL4K4_p6KkkBpSFQhjE0Jz-mlXpmnkASQ$) 

If you run into problems, please contact support by visiting [https://urldefense.com/v3/__https://github.com/contact__;!!OgnrLJOUc50AAzM!MhG52oeXB945a1KVue1Y16i78MJrqVlhH_8dv5hJiRTLOakL4K4_p6KkkBpSFQhjE0Jz-mlXzDXUPsE$](https://urldefense.com/v3/__https:/github.com/contact__;!!OgnrLJOUc50AAzM!MhG52oeXB945a1KVue1Y16i78MJrqVlhH_8dv5hJiRTLOakL4K4_p6KkkBpSFQhjE0Jz-mlXzDXUPsE$) 

Thanks,
The GitHub Team
my old user name is 124688DR  and my new one is 124688Droy

**Comment by LoisAlfred** [comment]
Su cuenta ha sido marcada.
Por eso, su perfil está oculto al público. Si cree que se trata de un error, póngase en [contacto con el servicio de asistencia](https://github.com/contact) para que revisen el estado de su cuenta.


ayudaaaaaaaa!!!!!!! ME SALE HACI
